kweb/Caddyfile

```diff
@@ -0,0 +1,30 @@
+*, http://, https:// {
+    gzip
+
+    tls self_signed
```

Comment:
the `self_signed` value should be coming from the environment (use the email, or if no host is set, use self_signed)